index.bs

```diff
@@ -210,6 +215,7 @@ Identify an unstable element {#sec-identify-unstable-element}
     When asked to <dfn export>identify an unstable element</dfn>, given <var>element</var> as input, run the following steps:
 
     1. If <var>element</var>’s <a>previous frame starting point</a> is null, return false.
+    1. If a <a>mousedown</a>, <a>keydown</a>, or <a>pointerup</a> user input event has happened within 500ms of the <a>current high resolution time</a>, return false.  
```

Comment:
Can we make this more clear? I would say this requires keeping track of 'last important event time' and then here we can compare with the current time. But even if we omit this and assume that the information is known, it's not clear which timestamp is being used for when those events 'happened'. Is it the event.timeStamp?